addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -380,17 +364,12 @@ public void packetReceived(byte[] packet) {
                 } else if (message instanceof RFXComTransmitterMessage) {
                     RFXComTransmitterMessage resp = (RFXComTransmitterMessage) message;
 
-                    byte seqNbr = getSeqNumber();
-                    if (resp.seqNbr == seqNbr) {
-                        logger.debug("Transmitter response received: {}", message.toString());
-                        setResponseMessage(resp);
-                        synchronized (notifierObject) {
-                            notifierObject.notify();
-                        }
-                    } else {
-                        logger.warn("Sequence number '{}' does not match, expecting number '{}'", resp.seqNbr, seqNbr);
-                    }
+                    logger.debug("Transmitter response received: {}", resp.toString());
 
+                    if (resp.response != Response.NAK)
```

Comment:
Minor: I prefer to have all the curly braces present but I don't believe it is an official part of our coding guidelines.